### DISCUSSION TITLE
Failed to find executables after compilation

### DISCUSSION THREAD

**Discussion by PulsEternal** [deepmodeling/deepmd-kit#399] [Q&A] [answered]
I compiled deepmd-kit-1.3.2 according to the[ installation guide](http://www.deepmd.org/download/installing-manual) with conda installed gpu version of tensorflow.  The Cmake and make process success except I only got dp_ipi in bin/ at last.
dp_ipi seems working fine however **dp_train, dp_frz, dp_test and dp_mdnn is missing from the folder**.

During make process there are some deprecation warning like:
```
warning: CUB requires C++14. Please pass -std=c++14 to your compiler. Define CUB_IGNORE_DEPRECATED_CPP_DIALECT to suppress this message.
CUB_COMPILER_DEPRECATION(C++14, pass -std=c++14 to your compiler);
```
By adding "-std=c++14" to CMakeLists.txt resolve the warning but I still could not got excutables I need. Does anyone has ideas how to resolve the problem?

---------
Installed tensorflow environments in anaconda:
```
# packages in environment at /dependencies/anaconda/anaconda3/envs/rdkit:
#
# Name                    Version                   Build  Channel
libtensorflow_cc          2.3.0            gpu_cuda10.1_0    deepmodeling
tensorflow                2.2.0           gpu_py37h1a511ff_0
tensorflow-base           2.2.0           gpu_py37h8a81be8_0
tensorflow-estimator      2.2.0              pyh208ff02_0
tensorflow-gpu            2.2.0                h0d30ee6_0
```
And I'm working with CUDA-11.1.

**Comment by njzjz** [comment, marked as answer]
This guide is out-of-date. The url of the current guide is https://deepmd.readthedocs.io/en/v1.3.2/.

**Reply by PulsEternal** [reply]
Thanks for your timely reply, the installation exactly follows the new guide.
